### PR TITLE
finishing up the enikshay for fab deploy

### PR DESCRIPTION
**Pull request by javierwilson**

### CODE REVIEW

File: fab/environments.yml [hunk 1]
```diff
@@ -37,6 +37,110 @@ base:
...
+        num_processes: 1
+      CaseSearchToElasticsearchPillow:
+        num_processes: 4
+      kafka-ucr-static:
+        num_processes: 4
+      ReportCaseToElasticsearchPillow:
+        num_processes: 1
+      SqlSMSPillow:
+        num_processes: 1
+      UnknownUsersPillow:
```
Comment (benrudolph) [inline fab/environments.yml:129]: if this is just for nikshay, we probably won't need this

File: fab/fabfile.py [hunk 2]
```diff
@@ -203,6 +203,15 @@ def l10k():
 
 
 @task
+@task
```
Comment (benrudolph) [inline fab/fabfile.py]: nit: 2 `@task`

File: fab/inventory/enikshay [hunk 3]
```diff
@@ -12,13 +12,13 @@
 [formplayer0] # formplayer, riak
 172.25.2.7
 
-[formplayer1] # formplayer
+[formplayer1] # formplayer, riak
```
Comment (benrudolph) [inline fab/inventory/enikshay:15]: interesting that riak is going to be sharing resources. do you know how much RAM will be on that machine?

Comment (javierwilson) [reply fab/inventory/enikshay:15]: 12G, how much memory should we use for formplayer_memory ?

Comment (benrudolph) [reply fab/inventory/enikshay:15]: i think roughly splitting half for riak and half for formplayer like you did is likely fine. i just know nikshay heavily heavily uses formplayer. so far formplayer hasn't proven to be memory constrained

**Review by mkangia — APPROVED** [review]

File: fab/environments.yml (hunk 1 above)
Comment (mkangia) [reply fab/environments.yml:129]: @javierwilson i am not sure about removing this.
checking with @emord.

Comment (emord) [reply fab/environments.yml:129]: We removed it for ICDS. should be ok to remove here.

It's only useful for forms that get submitted without a username (like some custom call via the api or something) which I don't think will happen with custom projects.

**Pull request merged**